README.md

```diff
@@ -560,6 +560,9 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2020. [PDF
 *[Real-time high speed motion prediction using fast aperture-robust event-driven visual flow](https://doi.ieeecomputersociety.org/10.1109/TPAMI.2020.3010468)*,  
 IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2020. [PDF](https://arxiv.org/pdf/1811.11135)
 
+- <a name="pivezhandi2020parahist"></a>Pivezhandi, M., Jones, P. H., Zambreno, J.,  
```

Comment:
Thanks. Could you format the entry like other entries and not leave an empty line between entries? Otherwise github thinks that you are modifying the entire list of optical flow. Please check the preview before finalizing the commit.
Also, please fix an unbalanced italics: *